tdrs-backend/docker-compose.yml

```diff
@@ -42,19 +42,20 @@ services:
   web:
     restart: always
     environment:
-      - DB_USER=tdpuser
-      - DB_PASSWORD=something_secure
+      - AV_SCAN_URL=http://clamav-rest:9000/scan
+      - DB_HOST=postgres
       - DB_NAME=tdrs_test
+      - DB_PASSWORD=something_secure
       - DB_PORT=5432
-      - DB_HOST=postgres
-      - DJANGO_SETTINGS_MODULE=tdpservice.settings.local
-      - DJANGO_CONFIGURATION=Local
-      - AV_SCAN_URL=http://clamav-rest:9000/scan
+      - DB_USER=tdpuser
+      - DJANGO_CONFIGURATION=${DJANGO_CONFIGURATION:-Local}
+      - DJANGO_SECRET_KEY=${DJANGO_SECRET_KEY:-tdp-dev-insecure}
```

Comment:
We need a consistent default for the `DJANGO_SECRET_KEY` and only want to allow this default in local or CI contexts so it is explicitly not defined in settings.